playground/mocks/data/idp/idx/authenticator-verification-select-authenticator.json

```diff
@@ -608,7 +608,7 @@
         "key":"custom_app",
         "id":"pfd1bmoy4nH6heq2v0g4",
         "displayName":"Custom Push App",
-        "logoUri":"https://okta.oktacdn.com/assets/img/logos/custompushLogo.svg",
+        "logoUri":"https://cdn.okta1.com/bc/globalFileStoreRecord?id=gfs3sti6DQ7A9vS3h0g4",
```

Comment:
How about http://localhost:3000/img/logos/default.png (this will serve from playground)